scripts/postinstall.js

```diff
@@ -0,0 +1,300 @@
+/* eslint-disable @typescript-eslint/no-var-requires,
+    @typescript-eslint/explicit-function-return-type
+*/
+
+const fs = require('fs');
+const rimraf = require('rimraf');
+
+// get blocks json allowed strings
+
+let themesLocaleList = [
+  'en',
+  'sv',
+  'no',
+  'fr',
+  'de',
+  'es',
+  'ja',
+  'ko',
+];
+
+const targetTimeZones = [
+  'Europe/Paris',
+  'Europe/Oslo',
+  'Europe/Stockholm',
+  'America/New_York',
+  'America/Chicago',
+  'America/Los_Angeles',
+  'America/Mexico_City',
+  'Pacific/Auckland',
+];
+
+const packageName = 'timezone';
+
+// in this case, wherever npm i was called
+// init cwd is the filepath of the initiating command
+const dirPath = `${process.env.INIT_CWD}/node_modules/${packageName}/`;
+
+try {
+  // eslint-disable-next-line global-require,import/no-absolute-path
+  themesLocaleList = require('/opt/engine/bundle/src/blocks.json').localeList;
+  // console.log('locale list via ./src/blocks.json');
+  // console.log(themesLocaleList.toString());
+} catch (e) {
+  // console.log('installing locally', e);
+}
+
+// input: themes locale
+// output: timezone-compatible locale
+function mapThemesLocales(themesLocaleString) {
+  let locale = '';
+
+  // via localizeDateHelper
+  switch (themesLocaleString) {
+    case 'sv':
+      locale = 'sv_SE';
+      break;
+    case 'fr':
+      locale = 'fr_FR';
+      break;
+    case 'no':
+      locale = 'nb_NO';
+      break;
+    case 'de':
+      locale = 'de_DE';
+      break;
+    case 'es':
+      locale = 'es_ES';
+      break;
+    case 'ja':
+      locale = 'ja_JP';
+      break;
+    case 'ko':
+      locale = 'ko_KR';
+      break;
+    case 'en':
+    default:
+      locale = 'en_US';
```

Comment:
in the future, we should allow users to pass in locales like British English `en_GB`. if not recognized then pass in the locale. 

someone was wanting to use pt: 

```
  'pt_BR.js',
  'pt_PT.js',
```

as long as a client uses their own translations, their feature pack will fallback to that localelist

Reply:
should support this in the future for date locale and (in a different ticket) localeList and locale:

```          
 "dateLocalization": {
            "language": "en_GB",
```